fuzzers/coverage_source_based/fuzzer.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Integration code for source-based coverage builds."""
+
+import os
+
+from fuzzers import utils
+
+
+def build():
+    """Build benchmark."""
+    cflags = [
+        '-fprofile-instr-generate', '-fcoverage-mapping', '-stdlib=libstdc++'
```

Comment:
I'm surprised you need to link against libstdc++ for this?
Could you post one of the errors you see when you didn't do this?
(you can use gists.github.com if it is very long)

Reply:
It looks like this:
/usr/bin/ld: /usr/lib/libFuzzer.a(FuzzerFork.o): undefined reference to symbol '_ZNSt18condition_variable10notify_oneEv@@GLIBCXX_3.4.11'
//usr/lib/x86_64-linux-gnu/libstdc++.so.6: error adding symbols: DSO missing from command line

Reply:
OK, could you look at how libFuzzer is compiled in fuzzers/libfuzzer/builder.Dockerfile use that method and then try getting rid of libstdc++ here?

Reply:
Sure!